Application/EdFi.Ods.WebApi/log4net.development.config

```diff
@@ -27,6 +27,12 @@
   <logger name="NHibernate">
     <level value="WARN"/>
   </logger>
+  <logger name="EdFi.Ods.Common.Infrastructure.Interceptors">
```

Comment:
These loggers are needed to quiet down the DEBUG logging of SQL that has been added -- effectively providing a mechanism to turn them on only when needed.